explore-analyze/dashboards/using.md

```diff
@@ -161,31 +161,46 @@ Filter the data within a specified range of time.
 
 ### View the panel data and requests [download-csv]
 
-**View the data in visualizations and the requests that collect the data:**
+Viewing the details of all requests used to build a visualization and the resulting data helps you confirm that the visualization is showing the right data and that the requests are performing as expected.
+
+The available actions can vary depending on the panel you're inspecting. 
+
+#### View and download the data in visualizations
+
+This action is possible for all charts created using **Lens** or {{esql}}. It is not available on other types of panels, such as **Maps** or **Vega** visualizations.
 
 1. Open the panel menu and select **Inspect**.
-2. View and download the panel data.
+1. Open the **View** dropdown, then select **Data**.
+1. If you'd like to download the data, select **Download CSV**, then select the format type from the dropdown:
 
-    1. Open the **View** dropdown, then click **Data**.
-    2. Click **Download CSV**, then select the format type from the dropdown:
+    * **Formatted CSV**: Contains human-readable dates and numbers.
+    * **Raw CSV**: Best used for computer use.
 
-        * **Formatted CSV** — Contains human-readable dates and numbers.
-        * **Unformatted** — Best used for computer use.
+        When you download visualization panels with multiple layers, each layer produces a CSV file, and the file names contain the visualization and layer {{data-source}} names.
```

Comment:
```suggestion
        When you download a visualization panel with multiple layers, each layer produces a CSV file, and the file names contain the visualization and layer {{data-source}} names.
```
I think this should be singular since line 173 says "open the panel..."